libipl/p10/common.C

```diff
@@ -287,3 +287,29 @@ int ipl_set_sbe_state_all_sec(enum sbe_state state)
 	}
 	return ret;
 }
+
```

Comment:
JS: Can you add some documentation about what you are doing with this API? It helps for future reference.
Looks, just adding only planar callout request and no deconfiguration/guard request.

Reply:
documentation is there in header file